docs/config/01-configuration-file.md

```diff
@@ -617,6 +617,44 @@ is handed off to [socket.io](http://socket.io/) (which manages the communication
 between browsers and the testing server).
 
 
+## upstreamProxy
+**Type:** Object
+
+**Default:** `undefined`
```

Comment:
Isn't the default he object containing the default values of the options listed below?